src/Server/src/Abstractions/Store/IResourceStore.cs

```diff
@@ -40,17 +35,12 @@ public interface IResourceStore<T> : IResourceStore
 
         Task<T> SaveAsync(T resource, CancellationToken cancellationToken);
 
-        new Task<IReadOnlyList<T>> GetResourceWithOpenApproval(
-            IEnumerable<Guid>? ids,
-            IEnumerable<string>? tenants,
-            CancellationToken cancellationToken);
-
         new Task<IReadOnlyList<T>> GetAllAsync(
             IEnumerable<Guid>? ids,
             IEnumerable<string>? tenants,
             CancellationToken cancellationToken);
 
-        Task<IReadOnlyList<T>> GetAllAsync(
+        new Task<IReadOnlyList<T>> GetAllAsync(
```

Comment:
couldn't we just call `GetAllAsync` with null as the arguments as the default implementation here?

Reply:
done